client/app/services/service-details/service-details.html

```diff
@@ -258,7 +258,7 @@ <h2 translate>Resources</h2>
                                     </div>
                                 </div>
                                 <div ng-if="!item.retired " class="list-view-pf-actions">
-                                    <div class="btn-group dropdown-kebab-pf" uib-dropdown dropdown-append-to-body
```

Comment:
Does this change belong here?

Seems like there's plenty more uses of `dropdown-kebab-pf`, including in the same file.